src/components/pages/Tutorial.vue

```diff
@@ -13,9 +13,10 @@
                 <div class="columnleft">
                     <h3>Find Task</h3>
                     <p>The goal of the Find task is to find and annotate high-risk objects.</p>
-                    <p>You are able to navigating through a bounded area in Google Street View. </p>
+                    <p>We define a high-risk object as an object that is touched by many different people (doorhandles, trash bins and park benches).</p>
```

Comment:
Slightly different wording maybe: "We define a high-risk object as an object that bight be touched by many different people..."